packages/gatsby-transformer-video/src/gatsby-node.js

```diff
@@ -0,0 +1,209 @@
+import { resolve, join } from "path"
+
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLInt,
+  GraphQLFloat,
+} from "gatsby/graphql"
+import { ensureDir } from "fs-extra"
+
+import FFMPEG from "./ffmpeg"
+
+exports.setFieldsOnGraphQLNodeType = (
+  { type, store },
+  { ffmpegPath, ffprobePath }
+) => {
```

Comment:
on the page [Building the GqlType](https://www.gatsbyjs.org/docs/schema-gql-type/) is an outdated notification:

> This documentation isn’t up to date with the latest [schema customization changes](https://www.gatsbyjs.org/docs/schema-customization).
>
> Outdated areas are:
> ...
> `setFieldsOnGraphQLNodeType` has been deprecated due to the new `createTypes` action
> ...

Reply:
Thanks will take this in to account

Reply:
done :)

Reply:
i think i can close this comment as resolved ...